src/client/app/graph/arrow/arrow.component.html

```diff
@@ -1,4 +1,7 @@
-<svg:g x="0" y="0">
+<svg:g x="0"
+       y="0"
```

Comment:
Let's go ahead and add the CSS rule "cursor: pointer" to the entire \<g\>

(to make it a bit easier to drag arrows)

Reply:
Done